spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -197,9 +197,15 @@ def __init__(self, resfile_path: PathType = None, clufile_path: PathType = None,
             assert folder_path.is_dir(), 'The folder_path must be a directory!'
 
             res_files = [f for f in folder_path.iterdir() if f.is_file()
-                         and '.res' in f.suffixes and '.temp' not in f.suffixes and len(f.suffixes) <= 2]
+                         and '.res' in f.suffixes 
+                         and '.temp' not in f.suffixes
```

Comment:
```suggestion
                         and '.res' in f.suffixes 
```

Reply:
You can remove the 'temp' option here since the len()==1 negates it

Reply:
Strange, I'm of course fine with the change, but can't commit this suggestion because GitHub tells me there would be no changes to the code.

Reply:
You can modify it manually on your end of the code (I can't access your branch here) and commit that way